docs/csharp/tutorials/string-interpolation.md

```diff
@@ -1,122 +1,98 @@
 
-## Create the Application
+## How to specify a format string for an interpolated expression
 
-Now that you've installed all the tools, create a new .NET Core
-application. To use the command line generator, create a directory for your project, such as `interpolated`, and execute the following command in your favorite shell:
+You specify a format string that is supported by the type of the expression result by following the interpolated expression with a colon (":") and the format string:
 
 ```
-dotnet new console
+{<interpolatedExpression>:<formatString>}
 ```
 
-This command creates a barebones .NET Core project with a project file, *interpolated.csproj*, and a source code file, *Program.cs*. You will need to execute `dotnet restore` to restore the dependencies needed to compile this project.
+The following example shows how to specify standard and custom format strings for expressions that produce date and time or numeric results:
 
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
+[!code-csharp-interactive[format string example](~/samples/snippets/csharp/tutorials/string-interpolation/Program.cs#2)]
 
-To execute the program, use `dotnet run`. You should see "Hello, World" output to the console.
+For more information, see the [Format String Component](../../standard/base-types/composite-formatting.md#format-string-component) section of the [Composite Formatting](../../standard/base-types/composite-formatting.md) topic. That section provides links to the topics that describe standard and custom format strings supported by .NET base types.
 
+## How to control the field width and alignment of the formatted interpolated expression
 
+You specify the minimum field width in which string representation of the expression result is inserted and the text alignment within that field by following the interpolated expression with a comma (",") and the constant expression that defines alignment:
 
-## Intro to String Interpolation
+```
+{<interpolatedExpression>,<alignment>}
+```
 
-With <xref:System.String.Format%2A?displayProperty=nameWithType>, you specify "placeholders" in a string that are replaced by the arguments following the string. For instance:
+If the alignment value is positive, the formatted string representation of the expression result is right-aligned; if negative, it's left-aligned.
```

Comment:
alignment can be italicized 
perhaps instead of "the formatted string representation of the expression result", just "the expression result"

Reply:
I've phrased 
> ...the formatted string representation of the expression result...

to stress the fact that the expression result is first formatted to text and only then alignment is applied.

Reply:
OK, that's fine.